package.json

```diff
@@ -45,7 +45,7 @@
   },
   "devDependencies": {
     "grunt": "^1.0.1",
-    "grunt-eslint": "^19.0.0"
+    "grunt-eslint": "^18.0.0"
```

Comment:
The eslint task is not ES5 compatible in the 19.0 and it does not work in node 0.10.
If the eslint task could be skipped when running on older node versions this wouldn't be a problem, but I think this is the easiest solution.